File: QRCode.FEW/ClientApp/src/app/models/qr_survey.ts
export interface qr_survey {
    qrsurveyid: number;
    name: string;
    code: string;
    status: boolean;
    additional: string;
    start_date: Date;
    end_date: Date;
    created_date: Date;
    created_by: number;
    lastcreated_date: Date;
    lastcreated_by: number;
}

File: QRCode.Core/Domain/qr_survey.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace QRCode.Core.Domain
{
    public class qr_survey
    {
        public virtual int qrsurveyid { get; set; }
        public virtual string name { get; set; }
        public virtual string code { get; set; }
        public virtual bool status { get; set; }
        public virtual string additional { get; set; }
        public virtual DateTime? start_date { get; set; }
        public virtual DateTime? end_date { get; set; }
        public virtual DateTime? created_date { get; set; }
        public virtual int created_by { get; set; }
        public virtual DateTime? lastcreated_date { get; set; }
        public virtual int lastcreated_by { get; set; }
    }
}
